readme.md

```diff
@@ -7,10 +7,12 @@
 - Ansible 2.x (http://docs.ansible.com/ansible/intro_installation.html)
 - Vagrant (https://www.vagrantup.com/docs/installation/)
 - NFS
+- Yarn
```

Comment:
The guest environment should not have a need for yarn.

Reply:
Removed.